### DISCUSSION TITLE
Search does not find text in big files

### DISCUSSION THREAD

**Discussion by aquinlivan** [community/community#39043] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Cannot search for text in a large text file.

**Comment by look** [comment, marked as answer]
There is a [350 KiB limit](https://docs.github.com/en/search-github/github-code-search/about-github-code-search#limitations) for documents to be indexed.

Can you let us know how big the file you're trying to search is, and what type it is (e.g., C++, Markdown, etc.). Thanks!

**Reply by aquinlivan** [reply]
This file in particular is 416 KB and it's an Oracle PL/SQL package (text). I have seen packages and code sets in the multiple MB size before so a search in larger files would be very useful.
For another example we have a code file that's almost 9 MB (export from Oracle Apex) and I am sure there a bigger ones.

**Reply by look** [reply]
Thanks for the context. I believe in the non-beta code search, large files are truncated rather than excluded entirely, which is why they show up in search (though it's only the first part of the file that's indexed and searchable.)
